content/pages/meetup/2023/0726-nycu.rst

```diff
@@ -0,0 +1,85 @@
+========================================
+Meetup 2023 July 26th in NYCU
+========================================
+
+:date: 2023-07-20 13:40
+:url: meetup/2023/0726-nycu
+:save_as: meetup/2023/0726-nycu.html
+
+Date
+-----
+
+* Date: July 26th, 2023
+* Time: 18:30 -- 21:30 (3 hours)
+
+|
+
+Agenda
+--------
+
+* 18:30-19:30 Free discussion
+* 19:40-20:10 Subject (sharing): Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis
+* 20:10-21:30 Subject (discussion): Preparations for August scisprint  
+
+|
+
+Subjects
+------------------
+
+* **Sharing: Physical Therapy Assessment with AI - Depth Camera for Mobility Analysis**
+
+    Speaker : Nian-Cheng - A graduate student in NYCU Institute of Artificial Intelligence Innovation.
+    
+    In this topic, I leveraged data from depth cameras for AI model data processing and introduces the essential tools I utilized for this purpose.
+
+* **Discussion: Preparations for August scisprint**
+
+    Additionally, we will discuss the following items for preparing August scisprint. 
+
+    * Task arrangement for volunteers
+    * Promotion plan for August scisprint
+    * Preparations for career conversation
+
+|
+
+Sign up
+------------
+
+The event is free. Please register on `sign up <https://sciwork.kktix.cc/events/meetup-20230726>`__.
+
+|
+
+About Meetup
+------------
+
+Meetup is an event providing space for people to work on open source
+projects together. We welcome any subjects that may interest the attendees,
+and especially encourage code for science, engineering, and technology, which
+demand more critical discussions than other applications of computer
+programming.
+
+We would like to provide a supportive and friendly environment for all 
+attendees to support more developers to join in the open-source communities. 
+
+To join the meetup, please bring your laptop. You are also very welcome to 
```

Comment:
Please add some text tot tell the attendees to sign up at kktix.

You can reference in [scisprint May webpage](https://github.com/sciwork/swportal/blob/a74570cf32bc5cd6b0d4341e83ee003333821d65/content/pages/sprint/2023/05-hsinchu.rst?plain=1#L30).